draft-ietf-ppm-dap.md

```diff
@@ -1876,6 +1908,10 @@ following checks:
 1. Check that the input share can be decoded as specified by the VDAF. If not,
    the input share MUST be marked as invalid with the error `invalid_message`.
 
+1. Check that the report's timestamp is well-formed as specified in
+   {{timestamps}}. If not, the Aggregator MUST mark the input share as invalid
+   with error `invalid_message`.
```

Comment:
Do we want a new report error variant here?

Reply:
I don't think so. It's hard to imagine any client out there that would first try an unrounded timestamp, then gets error `your_timestamp_sucks` and then goes "OK, FINE, I'll round it down!" Transmitting an unrounded timestamp in a report just suggests a defective client that isn't ever going to succeed.